app/ui-react/packages/ui/package.json

```diff
@@ -46,21 +46,25 @@
     "jest": "^25.1.0",
     "jest-cli": "^25.1.0",
     "microbundle": "^0.11.0",
+    "monaco-editor": "^0.19.3",
+    "monaco-editor-webpack-plugin": "^1.8.2",
+    "patternfly-react": "^2.34.3",
```

Comment:
So, again, this needs to go.  You need to swap out whatever you're using this for with the PF4 equivalent.